usecases/user/open-chat.md

```diff
@@ -67,12 +64,21 @@
         <td>
             <ul>
                 <li>
-                    Item 1
+                    2| Actor is not longer registered to the system
                 </li>
                 <li>
-                    Item 2
+                    2| Actor has deleted chat history
```

Comment:
this exception cannot occur as there is a precondition stating that the chat must be available